### DISCUSSION TITLE
more than 10 seconds to get rtsp from mediamtx?

### DISCUSSION THREAD

**Discussion by Gogh-123** [bluenviron/mediamtx#2839] [Questions]
### Question

I use opencv + gstreamer to push the rtsp, why does it take more than 10 seconds to establish a link when pulling the stream? Does the server need to set some parameters, or is there a problem with the parameters of the push stream?

**Comment by aler9** [comment]
Please reopen the issue and provide details that allow to replicate the issue, including:
- how are you publishing to the server and with which software
- how are you reading from the server and with which protocol and software
- server logs with "logLevel" set to "debug", as explicitly mentioned in the instructions

Otherwise it's impossible to understand what's going on.
